parser/src/ast.rs

```diff
@@ -359,7 +366,7 @@ impl Expression {
             | String {
                 value: FormattedValue { .. },
             } => "f-string expression",
-            Identifier { .. } => "named expression",
```

Comment:
This was a typo, right?  Or maybe it predates "named expressions" becoming an actual thing in Python?